Documentation/Storage-Configuration/NFS/nfs-csi-driver.md

```diff
@@ -207,3 +207,16 @@ To clean your cluster of the resources created by this example, run the followin
 ```console
 kubectl delete -f deploy/examples/csi/nfs/pvc-clone.yaml
 ```
+
+## Consuming NFS from an external source
+
+For consuming NFS services and exports external to the Kubernetes cluster (including those backed by an external standalone Ceph cluster), Rook recommends using Kubernetes' regular NFS consumption model. This requires the Ceph admin to create the needed export, while reducing the privileges needed in the client cluster for the NFS volume.
```

Comment:
`Kubernetes'` to `Kubernetes`

Reply:
`Kubernetes'` was actually correct because it should be possessive.

Reply:
@parth-gr ^

Reply:
If the possessive is confusing, it could be reworded to `Rook recommends using the regular NFS consumption model for Kubernetes`